coalib/testing/LocalBearTestHelper.py

```diff
@@ -193,6 +193,9 @@ def check_results(self,
         :param lines:            The lines to check. (List of strings)
         :param results:          The expected list of results.
         :param filename:         The filename, if it matters.
+        :param check_order:      Whether to check that the elements of results
```

Comment:
If you refer to the parameter (and I think you do), you should put `results` into double-backticks :+1: